app/models/manageiq/providers/nutanix/inventory/collector/infra_manager.rb

```diff
@@ -43,18 +43,18 @@ def hosts
     @hosts
   end
 
-  def volume_connection
-    @volume_connection ||= begin
-      config = NutanixVolumes::Configuration.new.tap do |c|
+  def cluster_mgmt_connection
+    @cluster_mgmt_connection ||= begin
+      config = NutanixClustermgmt::Configuration.new.tap do |c|
         c.scheme = "https"
         c.host = "#{manager.hostname}:#{manager.port || 9440}"
         c.username = manager.authentication_userid
         c.password = manager.authentication_password
-        c.verify_ssl = false
+        c.verify_ssl = OpenSSL::SSL::VERIFY_NONE
         c.verify_ssl_host = false
```

Comment:
We should be using the `manager.default_endpoint.verify_ssl` property here e.g. `verify_ssl_bool = manager.default_endpoint.verify_ssl == OpenSSL::SSL::VERIFY_PEER`